app/models/journal.rb

```diff
@@ -38,9 +38,6 @@ class Journal < ActiveRecord::Base
   register_journal_formatter :attachment, OpenProject::JournalFormatter::Attachment
   register_journal_formatter :custom_field, OpenProject::JournalFormatter::CustomField
 
-  # Make sure each journaled model instance only has unique version ids
-  validates_uniqueness_of :version, scope: [:journable_id, :journable_type]
```

Comment:
Does it do any harm to keep this just to be extra sure?

Reply:
There is a unique constraint on the SQL side so this is not really needed (an exception would be thrown in its stead) but yeah I don't mind